cli/autocomplete.py

```diff
@@ -189,8 +189,24 @@
             '--pea-id',
             '--pea-role',
             '--noblock-on-start',
-            '--dynamic-routing',
+            '--uses-before',
```

Comment:
This  is wrong, no uses before for gateway. Or parallel , shards and more

Reply:
If we say the gateway is a pod, it should have all features of a pod

Reply:
Right now this is not true, when this is true it may happen. But not now.

Reply:
true, I changed it. Thanks